docs/troubleshooting.md

```diff
@@ -186,13 +186,13 @@ stack install haskell-language-server --stack-yaml=stack-${ghcVersion}.yaml
 You also can leverage `ghcup compile hls`:
 
 ```bash
-ghcup compile hls -v 1.6.1.0 --ghc 8.10.7
+cabal update && ghcup compile hls -v 1.6.1.0 --ghc 8.10.7
```

Comment:
Can we maybe add a comment explaining this? Cabal update can take some time to run, and people don't have to execute it *every* time.